src/release/breaking-changes/android-14-nonlinear-text-scaling.md

```diff
@@ -0,0 +1,355 @@
+---
+title: Deprecate `textScaleFactor` in favor of `TextScaler`
+description: The `textScaleFactor` scalar is replaced by `TextScaler` in preparation for Android 14 nonlinear text scaling support.
+---
+
+## Summary
+
+In preparation for adopting the [Android 14 nonlinear font scaling][] feature, 
+all occurrances of `textScaleFactor` in the Flutter framework have been 
+deprecated and replaced by `TextScaler`.
+
+## Context
+
+Many platforms allow users to scale up / down textual contents globally in
+system preferences. In the past, the scaling strategy was captured as a single 
+`double` value named `textScaleFactor`, as text scaling was proportional: 
+`scaledFontSize = textScaleFactor x unScaledFontSize`. For example, When 
+`textScaleFactor` is 2.0 and the developer-specified font size is 14.0, the 
+actual font size is 2.0 x 14.0 = 28.0.
+
+With the introduction of [Android 14 nonlinear font scaling][], larger text gets
+scaled at a lesser rate as compared to smaller text, to prevent excessive scaling
+of text that is already large enough. The `textScaleFactor` scalar value used by 
+"proportional" scaling is not enough to represent this new scaling strategy.
+
+## Description of change
+
+Introducing a new interface `TextScaler`, which represents a text scaling strategy.
+
+```dart
+abstract class TextScaler { 
+  double scale(double fontSize);
+  double get textScaleFactor; // Deprecated. 
+}
+```
+
+The `scale` method should be used to scale font sizes in lieu of `textScaleFactor`.
+The `textScaleFactor` getter provides an estimated `textScaleFactor` value, for 
+backward compatibility purposes. It is already marked as "deprecated", but is 
+expected to have a longer lifetime than any other deprecations in this migration 
+guide.
+
+The new class has replaced 
+`double TextScaleFactor` (`double textScaleFactor` -> `TextScaleFactor textScaleFactor`),
+in the following APIs:
+
+#### `Painting` Library
```

Comment:
```suggestion
### Painting library
```
Skipping headers makes writers twitch... Also, we don't use text font in headers. And we use sentence case, not title case, for headers and titles.